### DISCUSSION TITLE
nextjs is inaccessible via yarn dev/npm run dev startup, help

### DISCUSSION THREAD

**Discussion by zacharydigital** [vercel/next.js#46228] [Help]
### According to the nextjs official documentation tutorial https://nextjs.org/docs/getting-started#automatic-setup after installing nextjs through the content of the tutorial guide to start found the following problems.

**The startup does not work with port addresses, some are accessible and some are not
What is the solution, help**

After running the npm run dev or yarn dev command, the service http://localhost:3001 is started, but this address can be accessed normally to start nextjs APP.

````
nextjs-blog [ main][ v19.6.1][💎 v3.0.0]
❯ yarn  dev
yarn run v1.22.19
warning ../../../package.json: No license field
$ next dev
warn  - Port 3000 is in use, trying 3001 instead.
ready - started server on 0.0.0.0:3001, url: http://localhost:3001
event - compiled client and server successfully in 203 ms (165 modules)
wait  - compiling / (client and server)...
event - compiled client and server successfully in 1089 ms (194 modules)
````
<img width="1179" alt="20230222-144831@2x" src="https://user-images.githubusercontent.com/7262553/220544438-d2d82546-e090-41ee-966e-1c0526421faa.png">

Launch next.js APP via terminal iterm, run npm run dev or yarn dev command and it prompts to start the service http://localhost:3000, but this address cannot be opened as a blank page

```
nextjs-blog [ main][ v19.6.1][💎 v3.0.0]
❯ yarn dev
yarn run v1.22.19
warning ../../../package.json: No license field
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
event - compiled client and server successfully in 762 ms (165 modules)
```

**Comment by icyJoseph** [comment]
If you run `pwd` on your terminal it should be something that ends with `nextjs-blog`, can you confirm that?

Asking because your command is printing `../../../package.json` that looks strange...

**Reply by zacharydigital** [reply]
why?

**Reply by zacharydigital** [reply]
The destination address for execution is here: `**/Users/zhenxing.z/project/next_demo/nextjs-blog/**`

**Reply by icyJoseph** [reply]
One additional question, what OS do you use? Catalina something something? I've observed before that certain older MacOS version's won't work at all with Next 13 + fonts (or just plain next 13 I think)

**Reply by zacharydigital** [reply]
macOS   ventura  13.0.1 (22A400)

**Comment by zacharydigital** [comment]
<img width="937" alt="image" src="https://user-images.githubusercontent.com/7262553/220616952-af934f6f-1e50-4799-893d-b0f5e7462c0d.png">
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/7262553/220617063-4ff8d98c-e559-429e-b386-f78526990d2d.png">

Cannot be accessed at http://localhost:3000/

**Reply by zacharydigital** [reply]
Why did it turn out this way, help. What should I do?  @leerob

**Reply by icyJoseph** [reply]
Do you have a repository to look at? I am still intrigued by the `../../../package.json`. Have you maybe installed next globally? or something like that? 

```console
yarn dev
yarn run v1.22.19
$ next dev
warn  - Port 3000 is in use, trying 3001 instead.
ready - started server on 0.0.0.0:3001, url: http://localhost:3001
warn  - You have enabled experimental feature (newNextLinkBehavior) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

event - compiled client and server successfully in 197 ms (171 modules)
```

When I run a project that's all I see.

Could you try to move to another directory and start fresh with:

```js
yarn create next-app delete-me
```

And see if that works? just like that? Also run `yarn global list` and make sure `next` is not listed there.

**Reply by zacharydigital** [reply]
ok, try

**Reply by icyJoseph** [reply]
Did it work out? This is such a weird scenario... If it doesn't work, could you make a repository in Github so we can see what's going on?
